docs/doc/90-contributing/08-how-to-write-scalar-functions.md

```diff
@@ -23,16 +23,12 @@ Scalar functions (sometimes referred to as User-Defined Functions / UDFs) return
 └─────┘                    └──────┘
 ```
 
-
 ### Knowledge before writing the eval function
```

Comment:
How to Write Scalar Functions  -> How to Write a S.. F..

Reply:
What's scalar functions -> What's Scalar Function

Reply:
A scalar function (also known as User-Defined Functions or UDFs) returns a single value for each row instead of a result set. Scalar functions can be used in most places within a query or SET statement (except the FROM clause).

Reply:
Knowledge before writing the eval function > What you need to know before writing